test/Graphics/Image.test.cpp

```diff
@@ -265,6 +265,20 @@ TEST_CASE("[Graphics] sf::Image")
             CHECK(!image.loadFromFile("."));
             CHECK(!image.loadFromFile("this/does/not/exist.jpg"));
 
+            std::array<char32_t, 10> utf32name = {'X', 'f', 'a', 'k', 'e', '.', 'p', 'n', 'g', '\0'};
+
+            utf32name[0] = 0xf1; // small n with tilde, outside of ascii but inside common latin1 codepage
+            CHECK(!image.loadFromFile(std::filesystem::path(utf32name.data())));
```

Comment:
```suggestion
            // small n with tilde, outside of ascii but inside common latin1 codepage
            CHECK(!image.loadFromFile(std::filesystem::path(U"\xF1fake.png")));
```
Because we can use string literals, we can just pass the string to the function rather than store it on the stack and give it a name.

Reply:
This is a different string, because your first char here is `\xF1fa`

Reply:
Yes, and `\x{}` and `\u{}` is from C++23. I used ASCII in code and arrays to be 100% sure no funny stuff like that happens.

Reply:
You could do something like `U"\xF1"  "fake.png"` (like what I did in godbolt) to try to ensure it, but that already loses the 'readability' advantage imo.

Reply:
Yes, or use `"missingX.png"`. I put X first so it's `arr[0] = 0xWhatever`, no need to think/find/remember offset, and used "fake" instead of "missing" or "does-not-exist" to make array init less typing. There's no real reason it's just 1 unicode char or that it's first, so with literal we can just put unicode last (before dot) and use more than 1 char per string too.

Reply:
I put in a whole other filename "missing" + unicode char + the extension (not that extension even matters when missing file).